app/controllers/idv/gpo_verify_controller.rb

```diff
@@ -50,7 +50,12 @@ def create
               disavowal_token: disavowal_token,
             )
             flash[:success] = t('account.index.verification.success')
-            redirect_to sign_up_completed_url
+
+            if result.extra[:threatmetrix_check_failed]
+              redirect_to idv_setup_errors_url
+            else
+              redirect_to sign_up_completed_url
+            end
```

Comment:
If TM check has failed here, we are still:

- Creating the `:account_verified` event
- Alerting the user that their account has been verified

I _think_ this is the desired behavior -- technically their account  _is_ verified, but they shouldn't be able to use it to do anything IAL2-related (see #7493)